backend/logger.py

```
@@ -0,0 +1,46 @@
+import logging
+import os
+from datetime import datetime
```

Ruff (0.11.9)

`datetime.datetime` imported but unused

Remove unused import: `datetime.datetime`

(F401)